src/lib/rotate.py

```diff
@@ -0,0 +1,19 @@
+""" swap table and function for rotating chars 180 degrees.
+"""
+
+swapTable = {
+    '(':')',
+    ')':'(',
+    '.':'`',
+    '`':'.',
+    '_':'^',
+    "'":','
+    }
+
+def rotate(char):
+    """ Rotate char if it has a "symmetric" char. 
+    """
+    if char in swapTable.keys():
+        return swapTable[char]
+    else:
+        return char
```

Comment:
You can simply write `return swapTable.get(char, char)` instead, it is the same.
Also you can use `translate` method of `string` here (or even where the `rotate` function is called).

https://www.programiz.com/python-programming/methods/string/translate